README.md

```diff
@@ -17,7 +17,7 @@ Remark42 is a self-hosted, lightweight, and simple (yet functional) comment engi
 * Images upload with drag-and-drop
 * Extractor for recent comments, cross-post
 * RSS for all comments and each post
-* Telegram and email notifications
+* Telegram, Slack and email notifications
```

Comment:
I think by now we should better add clarification that Telegram and Slack and email are available for admins, and email is the only one available to users at the moment. Would be nice if you would do that documentation change as well.